test/jdk/ProblemList.txt

```diff
@@ -797,3 +797,8 @@ java/awt/event/MouseEvent/SpuriousExitEnter/SpuriousExitEnter.java 8254841 macos
 java/awt/Focus/AppletInitialFocusTest/AppletInitialFocusTest1.java 8256289 windows-x64
 java/awt/FullScreen/TranslucentWindow/TranslucentWindow.java 8258103 linux-all
 java/awt/Focus/FrameMinimizeTest/FrameMinimizeTest.java 8016266 linux-x64
+
+############################################################################
+
+# Loom, fibers branch
+
```

Comment:
This is left over from when Stress.java was excluded so we can remove it from the PR.